plugins/origin_server_auth/origin_server_auth.cc

```diff
@@ -1087,9 +1100,12 @@ config_reloader(TSCont cont, TSEvent /* event ATS_UNUSED */, void *edata)
   S3Config *file_config = gConfCache.get(s3->conf_fname());
 
   if (!file_config || !file_config->valid()) {
-    TSError("[%s] invalid configuration. Check mandatory fields.", PLUGIN_NAME);
+    TSError("[%s] invalid configuration. Check mandatory fields. Scheduling reload", PLUGIN_NAME);
+    long delay = pow(s3->incr_invalid_file_count(), 2);
```

Comment:
Do you mean power of 2?  Please use bitwise left shift `1 << s3->incr_invalid_file_count()`.